import/popups/WebrtcPermissionDialog.qml

```diff
@@ -0,0 +1,174 @@
+/****************************************************************************
+**
+** Copyright (c) 2021 Open Mobile Platform LLC.
+**
+****************************************************************************/
+
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import QtQuick 2.0
+import Sailfish.Silica 1.0
+
+UserPromptDialog {
```

Comment:
This looks good to me.

Open question: what is your opinion of the extension mechanism / popup interfaces?  Are there pain points / things we should change now, before we ship?  Or was it fairly straightforward to use, in your opinion?

Reply:
At first glance, this is a little confusing, but after a thought, I can't figure out what could be improved.